boards/arm/stm32f4_disco/doc/index.rst

```diff
@@ -96,6 +96,10 @@ The Zephyr stm32f4_disco board configuration supports the following hardware fea
 +-----------+------------+-------------------------------------+
 | USB       | on-chip    | usb                                 |
 +-----------+------------+-------------------------------------+
+| CAN       | on-chip    | CAN controller                      |
++-----------+------------+-------------------------------------+
+
+.. note:: CAN feature requires CAN transceiver, such as `SK Pang CAN breakout board`_.
```

Comment:
Might be also interesting to state that CAN2 is the default IP and could not be enabled with CAN1

Reply:
@erwango Thanks, I expanded the note with those constraints.